.github/workflows/build-test.yml

```diff
@@ -42,20 +42,40 @@ jobs:
 
       - run: make format
 
-  goreleaser:
-    runs-on: ubuntu-latest
+  goreleaser-windows:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: msys2 {0}
     steps:
       - uses: actions/checkout@v3
+      - uses: msys2/setup-msys2@v2
         with:
-          fetch-depth: 0
+          msystem: UCRT64
+          update: true
+          install: git mingw-w64-ucrt-x86_64-gcc
       - uses: actions/setup-go@v4
         with:
           go-version-file: 'go.mod'
       - name: Install GoReleaser
         uses: goreleaser/goreleaser-action@v6
         with:
-          install-only: true
           version: latest
+          args: build --clean --snapshot --single-target --id bruin-windows-amd64
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          VERSION: ${{ github.ref_name }}
+
+
+  goreleaser:
```

Comment:
I'd call this `goreleaser-unix` to make it explicit

Reply:
done